steward/src/commands/schedule_corks/cellar_v1/set_fees_distributor.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{application::APP, cellars, cork, prelude::*};
+use abscissa_core::{clap::Parser, Command, Runnable};
+use ethers::abi::{AbiEncode, Address};
+use steward_abi::cellar::*;
+
+/// Fees Distributor subcommand
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nCalls setFeesDistributor() on the target cellar contract at the specified block height.\nFor more information see https://github.com/PeggyJV/cellar-contracts/blob/main/src/base/Cellar.sol"
+)]
+pub struct SetFeesDistributorCmd {
+    #[clap(short = 'n', long)]
+    /// Fee distributor's address
+    new_fees_distributor: Address,
```

Comment:
This is the ethers `Address` type, and the bytes have to come from a Cosmos address.